server/events/runtime/init_step_runner_test.go

```diff
@@ -57,10 +58,31 @@ func TestRun_UsesGetOrInitForRightVersion(t *testing.T) {
 				RepoRelDir: ".",
 			}, []string{"extra", "args"}, "/path")
 			Ok(t, err)
-			// Shouldn't return output since we don't print init output to PR.
+			// When there is no error, should not return init output to PR.
 			Equals(t, "", output)
 
 			terraform.VerifyWasCalledOnce().RunCommandWithVersion(logger, "/path", []string{c.expCmd, "-no-color", "extra", "args"}, tfVersion, "workspace")
 		})
 	}
 }
+
+func TestRun_ShowInitOutputOnError(t *testing.T) {
```

Comment:
@jeff-knurek thanks for the PR. The only thing I changed was to move this test case out of the one above. That one was testing that we switched between get and init so it didn't make sense to test getting output on error in that test case.